core/src/main/java/com/gentics/mesh/core/data/node/impl/NodeImpl.java

```diff
@@ -1985,6 +1986,22 @@ private PathSegment getSegment(String branchUuid, ContainerType type, String seg
 					return new PathSegmentImpl(container, binaryField, container.getLanguageTag(), segment);
 				}
 			}
+			//TODO S3
```

Comment:
Is there something left here to do?

Reply:
No, I had forgot to remove this. Done now :)